lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -348,14 +351,15 @@ class Method < Member
           location: T.any(Prism::Location, RubyIndexer::Location),
           name_location: T.any(Prism::Location, Location),
           comments: T::Array[String],
-          parameters: T::Array[Parameter],
+          overloads: T::Array[T::Array[Parameter]],
```

Comment:
We may want to introduce `Entry::Overload`. This would allow us to store Location against each overload.